src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -362,6 +373,13 @@ private void tryToAddVariant(final VariantContext vc, final ReferenceSequence re
                 final String refString = StringUtil.bytesToString(ref, vc.getStart() - 1, vc.getEnd() - vc.getStart() + 1);
 
                 if (!refString.equalsIgnoreCase(allele.getBaseString())) {
+                    // consider that the ref and the alt may have been swapped in a simple biallelic SNP
+                    if (vc.getAlleles().size() == 2 &&
+                            vc.getAlleles().stream().map(Allele::length).allMatch(l -> l == 1) &&
+                            refString.equalsIgnoreCase(vc.getAlleles().stream().filter(Allele::isNonReference).findFirst().get().getBaseString())) {
+                        sorter.add(LiftoverUtils.swapRefAlt(vc,TAGS_TO_REVERSE, TAGS_TO_DROP));
```

Comment:
I feel like there should be one more piece of logic here, but will give you the final say in overriding this.
If the original AF was 1.0 then this isn't really an allele swap; rather it's a reference fix and we should just remove the record (since no samples are polymorphic in the new reference).

Reply:
So, you would like the VC to be dropped if it turns out to be monomorphic after the allele swapping? My sense is that a monomorphic variant is easily removed post-hoc but that it could be useful for some to see them before being removed...

Reply:
Okay, so maybe we should keep it in but filter it?  Filter could be e.g. ReferenceFix.